server/data/prisonApiClient.test.ts

```diff
@@ -70,9 +70,6 @@ describe('prisonApiClient', () => {
           agencyName: 'Hewell (HMP)',
         },
         legalStatus: 'SENTENCED',
-        privilegeSummary: {
```

Comment:
Unrelated to this PR - consequence of updating the type definitions for all the APIs...